src/extension/loadLogs.ts

```diff
@@ -13,14 +13,23 @@ import { augmentLog, JsonMap } from '../shared';
 
 const driverlessRules = new Map<string, ReportingDescriptor>();
 
-export async function loadLogs(uris: Uri[], token?: { isCancellationRequested: boolean }) {
+function jsonParse(text: string, skipPointers?: 'skipPointers'): { data: Log, pointers: JsonMap | undefined } {
+    return skipPointers
+        ? {
+            data: JSON.parse(text),
+            pointers: undefined,
+        }
+        : jsonMap.parse(text) as { data: Log, pointers: JsonMap };
+}
+
+export async function loadLogs(uris: Uri[], skipPointers?: 'skipPointers', token?: { isCancellationRequested: boolean }) {
```

Comment:
Sorry what's the reasoning behind having an option for `skipPointers` flag? From what it seems like right now regardless of file size we won't be generating source maps right? 

Is it related to the question we asked to Roy about ability to having it on demand?

Reply:
I just read the email thread, it gave more context. Thanks, resolving this comment now